packages/react-core/src/helpers/PopoverBase/PopoverBase.tsx

```diff
@@ -1,44 +1,50 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import PropTypes from 'prop-types';
-import tippy from 'tippy.js';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import tippy, { Instance, Props, Content } from 'tippy.js';
+
+// eslint-disable-next-line @typescript-eslint/interface-name-prefix
+export interface ITippyProps {
+  /** Props for tippy */
+  [_: string]: any;
+}
+
+interface PopoverBaseProps extends ITippyProps {
```

Comment:
what about `hideOnClick`? is that already part of ITippyProps?

Reply:
`ITippyProps` takes any key mapped to any value. So, yes.